### DISCUSSION TITLE
SAE WCX Paper made with strictdoc

### DISCUSSION THREAD

**Discussion by BenGardiner** [strictdoc-project/strictdoc#1658] [Show and tell]
I'm happy to report that our paper made with strictdoc has been accepted to WCX 2024:
 https://www.sae.org/publications/technical-papers/content/2024-01-2806/

Thanks you @stanislaw , @mettta  and all strictdoc contributors!

**Comment by stanislaw** [comment]
Thanks a lot for sharing! We are super excited to see our tool being used!
